tests/rest/client/test_transactions.py

```diff
@@ -34,11 +38,13 @@ def setUp(self) -> None:
         self.hs.get_auth = Mock()
         self.cache = HttpTransactionCache(self.hs)
 
-        self.mock_http_response = (HTTPStatus.OK, "GOOD JOB!")
+        self.mock_http_response = (HTTPStatus.OK, {"result": "GOOD JOB!"})
```

Comment:
I guess this is just making the mock response match what we expect a `RestServlet` method to return?

Reply:
`fetch_or_execute` requires an `(int, JsonDict)` I believe. So yeah it seemed easiest to just give a more accurate value.